ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -555,7 +563,15 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
 
   const TransToAdmin = {
     hasAddedAdministratorsInGroup: (groupId): boolean =>
-      hasAddedPartyTypeToGroup(groupId, HomeOwnerPartyTypes.ADMINISTRATOR)
+      hasAddedPartyTypeToGroup(groupId, HomeOwnerPartyTypes.ADMINISTRATOR),
+    hasAtLeastOneAdminInGroup: (groupId): boolean => {
```

Comment:
Should add method add a more general method for `hasAtLeastOne...` to avoid code reputation.